pmd-plsql/etc/grammar/PldocAST.jjt

```diff
@@ -2904,9 +2906,9 @@ ASTTrimExpression TrimExpression() :
 {
   (
 	"(" { sb.append("(");}
-	 [ ( <LEADING> | <TRAILING> | <BOTH> ){ sb.append(" "); sb.append(token.toString()); }  ]
-	 [ simpleNode = StringExpression() { sb.append(" "); sb.append(simpleNode.getImage()); } ]
-	 ( <FROM>  )  { sb.append(token.image);}
+	 [ [ ( <LEADING> | <TRAILING> | <BOTH> ) { sb.append(" "); sb.append(token.toString()); }  ]
```

Comment:
With this change, PMD can't parse this snippet anymore:
https://github.com/pmd/pmd/blob/master/pmd-plsql/src/test/resources/net/sourceforge/pmd/lang/plsql/ast/TrimFunction.pls
([CI Build](https://travis-ci.org/pmd/pmd/jobs/527595158#L5395))

Can you verify, whether this file contains invalid syntax or whether the grammar has a problem?
Also please extend this file with an example call e.g. `TRIM(hire_date)`, that failed without your change and should be working now.